mappings/net/minecraft/world/World.mapping

```diff
@@ -282,15 +336,22 @@ CLASS net/minecraft/class_99 net/minecraft/world/World
 	METHOD method_357 getLightAtPos (Lnet/minecraft/class_106;Lnet/minecraft/class_1372;)I
 		ARG 1 type
 		ARG 2 pos
+	METHOD method_358 queueBlockEntity (Lnet/minecraft/class_348;)V
+		ARG 1 blockEntity
 	METHOD method_359 isRegionLoaded (Lnet/minecraft/class_476;Z)Z
 		ARG 1 box
+		ARG 2 canBeEmpty
 	METHOD method_360 hasEntityIn (Lnet/minecraft/class_646;)Z
-		ARG 1 box
-	METHOD method_361 containsBlockWithMaterial (Lnet/minecraft/class_646;Lnet/minecraft/class_591;)Z
 		ARG 1 area
```

Comment:
while you're at it, could you replace this as well?

Reply:
all boxes should be fixed now

Reply:
alright thanks